debug_toolbar/settings.py

```diff
@@ -40,6 +40,7 @@
     "SKIP_TEMPLATE_PREFIXES": ("django/forms/widgets/", "admin/widgets/"),
     "SQL_WARNING_THRESHOLD": 500,  # milliseconds
     "OBSERVE_REQUEST_CALLBACK": "debug_toolbar.toolbar.observe_request",
+    "TOOLBAR_LANGUAGE": None,
```

Comment:
I'm torn between having this be LANGUAGE or TOOLBAR_LANGUAGE. Does anyone have a strong opinions?

Reply:
I think both are fine, or even `LANGUAGE_CODE` (same as `settings.LANGUAGE_CODE`); the `TOOLBAR_` prefix is a bit redundant since all toolbar settings mostly affect the toolbar.

No strong opinions :)